go.mod

```diff
@@ -4,128 +4,159 @@ go 1.17
 
 require (
 	github.com/celestiaorg/nmt v0.8.0
-	github.com/cosmos/cosmos-sdk v0.44.3
-	github.com/cosmos/ibc-go v1.2.2
+	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20220418184507-c53157dd63f6
```

Comment:
There is no tagged release here yet?

Reply:
Still waiting on this one

Reply:
whoops, thanks for the reminder!! https://github.com/celestiaorg/celestia-app/pull/399/commits/17379f74ca7178e5bc0dfe0d284ad1903fc25c57